Community/bulk/bulk.py

```diff
@@ -0,0 +1,114 @@
+# Copyright 2019 BlueCat Networks (USA) Inc. and its affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+from bluecat_portal.customizations.tools import tools
```

Comment:
I think this needs to be updated to import the tools module from the workflow folder.

Reply:
@chrisjws do this need to be updated to use /customizations in the Community directory?

Reply:
So either the tools module needs to go into the Community directory and instructions are added, or the import should import from the workflow directory.